props/jetpack_Oli_buttons.h

```diff
@@ -0,0 +1,931 @@
+  // Proffie board in it can't realisticly have twist or stab motion events.
+  bool Event2(enum BUTTON button, EVENT event, uint32_t modifiers) override {
+    switch (EVENTID(button, event, modifiers)) {
+#ifdef BLADE_DETECT_PIN
+      case EVENTID(BUTTON_BLADE_DETECT, EVENT_LATCH_ON, MODE_ANY_BUTTON | MODE_ON):
+      case EVENTID(BUTTON_BLADE_DETECT, EVENT_LATCH_ON, MODE_ANY_BUTTON | MODE_OFF):
+        blade_detected_ = true;
+        FindBladeAgain();
+        SaberBase::DoBladeDetect(true);
+        return true;
+
+      case EVENTID(BUTTON_BLADE_DETECT, EVENT_LATCH_OFF, MODE_ANY_BUTTON | MODE_ON):
+      case EVENTID(BUTTON_BLADE_DETECT, EVENT_LATCH_OFF, MODE_ANY_BUTTON | MODE_OFF):
+        blade_detected_ = false;
+        FindBladeAgain();
+        SaberBase::DoBladeDetect(false);
+        return true;
+#endif // BLADE_DETECT_PIN
+      // Jetpack ON AND Volume Up:
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_ON):
+        if (!mode_volume_) {
+          if       (idle_) StartFlightMode(); // Transition from idle to flight  (rev more up)
+          else if (flight_) StopFlightMode(); // Transition from flight to idle     (rev down)
+          else               StartIdleMode(); // Jetpack initial start from 0 to idle (rev up)
+          // if idle, it revs up / if up, it revs down to idle / if not idle or not up, it needs to rev "up" to idle.
+        } else VolumeUp();
+        return true;
+      // Completely turn jetpack off from any state ******** only for 2 & 3 buttons (while on): ********
+      case EVENTID(BUTTON_AUX2, EVENT_FIRST_HELD_LONG, MODE_ON):
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_HELD_LONG, MODE_ON):
+        StopIdleMode(); // This will stop the jetpack but not the missile.
+        return true;
+      // Jetpack mishaps:
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_MEDIUM, MODE_OFF):
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_MEDIUM, MODE_ON):
+        JetpackMishaps();
+        return true;
+      // Missile Launch:
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_ON):
+        missilelaunch_ = true;
+        return true;
+
+      // Abort missile sequence, maybe? ******** only for (2 & 3 buttons while off): ********
+      case EVENTID(BUTTON_AUX2, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+        missilelaunch_ = false;
+        state_machine_.reset_state_machine();
+        PVLOG_NORMAL << "Missile Sequence Aborted.\n";
+        SaberBase::DoEffect(EFFECT_DISARM,0);
+        return true;
+
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+      // Next Preset (1 button):
+#if NUM_BUTTONS == 1
+        if (!mode_volume_) next_preset();
+        return true;

```

Comment:
Since you have return true at the end of both the #if and the #else, you should just move it out to after the #endif.